examples/sagaMonitor/index.js

```diff
@@ -181,7 +180,14 @@ function logSimpleEffect(effect) {
 function getEffectLog(effect) {
   let data, log
 
-  if(data = asEffect.take(effect.effect)) {
+  if(effect.root) {
+    data = effect.effect
+    log = getLogPrefix('run', effect)
+    log.formatter.addCall(data.saga.name, data.args)
+    logResult(effect, log.formatter)
+  }
+
```

Comment:
kinda weird this `else if` being put after a whiteline, no place in the code follows that pattern (i think)